.github/workflows/test-notebooks.yml

```diff
@@ -0,0 +1,25 @@
+name: Test notebooks
+
+on: [push,pull_request]
+
+jobs:
+  nbmake:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-22.04, macos-12, windows-2022]
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
```

Comment:
Thanks for the effort Matt. We've bumped `actions/setup-python@v4` across several repositories in the last few days.

Eventually, this workflow could also be transferred to `ITKRemoteModuleBuildTestPackageAction` @tbirdso ?

Reply:
Fully in favor of this 👍

Reply:
Thanks for the review @jhlegarreta !

Done.

I like the idea of consolidating the configuration.

Reply:
Resolving.